python/oneflow/nn/module.py

```diff
@@ -254,47 +333,229 @@ def _named_members(self, get_members_fn, prefix="", recurse=True):
+            ...    if name in ['bias']: # doctest: +SKIP
+            ...        print(param.size()) # doctest: +SKIP
+
+        """
         gen = self._named_members(
             lambda module: module._parameters.items(), prefix=prefix, recurse=recurse
         )
         for elem in gen:
             yield elem
 
     def buffers(self, recurse: bool = True) -> Iterator[Tensor]:
+        r"""
+        buffers(recurse=True) -> Iterator[Tensor]
+        
+        Returns an iterator over module buffers.
+
+        Args:
+            recurse (bool): if True, then yields buffers of this module
+                and all submodules. Otherwise, yields only buffers that
+                are direct members of this module.
+
+        Yields:
+            oneflow.Tensor: module buffer
+
+        Example::
+
+            >>> for buf in model.buffers(): # doctest: +SKIP
+            ...     print(type(buf), buf.size()) # doctest: +SKIP
+            <class 'oneflow.Tensor'> oneflow.Size([10])
+
+        """
         for (name, buf) in self.named_buffers(recurse=recurse):
             yield buf
 
     def named_buffers(
         self, prefix: str = "", recurse: bool = True
     ) -> Iterator[Tuple[str, Tensor]]:
+        r"""
+        named_buffers(prefix="", recurse=True) -> Iterator[Tuple[str, Tensor]]
+        
+        Returns an iterator over module buffers, yielding both the
+        name of the buffer as well as the buffer itself.
+
+        Args:
+            prefix (str): prefix to prepend to all buffer names.
+            recurse (bool): if True, then yields buffers of this module
+                and all submodules. Otherwise, yields only buffers that
+                are direct members of this module.
+
+        Yields:
+            (string, oneflow.Tensor): Tuple containing the name and buffer
+
+        Example::
+
+            >>> for name, buf in self.named_buffers(): # doctest: +SKIP
+            ...    if name in ['running_var']: # doctest: +SKIP
+            ...        print(buf.size()) # doctest: +SKIP
+
+        """
         gen = self._named_members(
             lambda module: module._buffers.items(), prefix=prefix, recurse=recurse
         )
         for elem in gen:
             yield elem
 
     def children(self) -> Iterator["Module"]:
+        r"""
+        children() -> Iterator["Module"]
+        
+        Returns an iterator over immediate children modules.
+
+        Yields:
+            Module: a child module
+            
+        Example::
+
+            >>> import oneflow.nn as nn
+            >>> l1 = nn.Linear(2, 2)
+            >>> l2 = nn.Linear(2, 2)
+            >>> net = nn.Sequential(l1, l2)
+            >>> for idx, m in enumerate(net.children()): # doctest: +SKIP
+            ...     print(idx, '->', m) # doctest: +SKIP
+
+            0 -> Linear(in_features=2, out_features=2, bias=True)
+            1 -> Linear(in_features=2, out_features=2, bias=True)
```

Comment:
```suggestion
            >>> for idx, m in enumerate(net.children()):
            ...     print(idx, '->', m)
            0 -> Linear(in_features=2, out_features=2, bias=True)
            1 -> Linear(in_features=2, out_features=2, bias=True)
```

这里是真实代码及输出，是可以不用 SKIP 的吧。